buildenv/jenkins/JenkinsfileBase

```diff
@@ -144,8 +144,17 @@ def test() {
 		}
 		stage('Test') {
 			timestamps{
-				echo 'Running tests...'
-				sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _$TARGET"
+				if (env.BUILD_LIST == "jck" && env.SPEC == "linux_x86-64"){
+					wrap([$class: 'Xvfb', displayName: 2]) {
+						withEnv(["DISPLAY=:2"]) {
```

Comment:
I had understood if it was not set, as in if you use 
wrap([$class: 'Xvfb']) 
it would auto-number

Best check the xvfb plugin documentation, as it is preferred to not hard-code if can be avoided.

Reply:
X11 (or Xvfb) related tests need environment variable `DISPLAY=target_xvfb_monitor` to be set. The `DISPLAY` is not a default system environment variable, so it needs to be set manually according to the Xvfb display number.

`wrap([$class: 'Xvfb'])` or `wrap([$class: 'Xvfb', autoDisplayName: true])` will automatically initialize the Xvfb with a random available display number. At this time, the `DISPLAY` needs to be set the same as the display number. 

Updated the JenkinsfileBase to grab the Xvfb display number and set the DISPLAY.

Reply:
Turns out that if there are multiple Xvfb process running, the `ps -ax  | grep '[X]vfb' | awk '{print $6}'` shell cmd will return all of them, which will trigger test failures. `ps -a ` or `ps` will not print out the display number. I think the safe way to use Xvfb now is to define specific number and export DISPLAY to that number. Any suggestions?

Reply:
did a search and seems `ps -f` will do the expectation. the PR has been updated.